### ISSUE TITLE
Logo for Span<T> should be Spanatee the Manatee

### ISSUE THREAD

**Issue by Eilon** [dotnet/corefxlab#1034]
It has become clear that the new `Span<T>` type needs a logo. Every good type should have a logo.

I have two highly artistic ideas to toss around for the official logo of the `Span<T>` type. It revolves around the character of Spanatee the [Manatee](https://en.wikipedia.org/wiki/Manatee) (it rhymes, and "Spanatee" sounds like "Span of T").

I will post each idea as its own comment so that people can vote on it. Feel free to post additional ideas (one per comment) for voting. This is not final artwork; we would presumably hire a world-renowned artist to do the final artwork.

**Comment by Eilon** [comment]
Idea 1: Bracketed manatees showing the generic shape of `<T>`:

![image](https://cloud.githubusercontent.com/assets/202643/20698557/af996778-b5b6-11e6-996a-e3b322d4775a.png)

**Comment by Eilon** [comment]
Idea 2: Spanatee the Manatee him/herself with `<T>` on its body:

![image](https://cloud.githubusercontent.com/assets/202643/20698552/a8f24638-b5b6-11e6-9e88-cfd99569f9b8.png)

**Comment by JimBobSquarePants** [comment]
![spanatee](https://cloud.githubusercontent.com/assets/385879/20699103/9f23c034-b659-11e6-8912-80cae61aa195.png)

**Comment by jstclair** [comment]
Clearly should be a manatee drinking a cup of tea. Preferably with a top hat and monocle

**Comment by JvanderStad** [comment]
Clearly we should ask Roland Tamayo if we may use his manatee as a mascotte for Span&lt;T&gt;

![image](https://cloud.githubusercontent.com/assets/5426370/20715530/c8ed3e72-b64f-11e6-91df-d80193d40058.png)

(Source: https://www.instagram.com/p/BB8L6jnQDIo/)

**Comment by KrzysztofCwalina** [comment]
This one even has "the cloud" in the background. Though what's with all the pollution? Shouldn't it be kittens and flowers be coming out of the ~~chimneys~~piplelines.

**Comment by JvanderStad** [comment]
Maybe it's a visual representation of the new System.IO.Pipelines namespace? (https://github.com/dotnet/corefxlab/pull/980).. +1 on the kittens & flowers

**Comment by benaadams** [comment]
>  Though, what's up with all the pollution

Isn't pollution, its the air being "on fire" due to the high speed...

**Comment by KrzysztofCwalina** [comment]
"Span<T>: the 8th wonder of the 21st century. It makes even manatees fly really really fast."

**Comment by Drawaes** [comment]
Unless there is more bounds checks... then they lose speed and can't
generate enough lift and die... wont someone please think of the poor
Manatees!

**Comment by joshfree** [comment]
nice mspaint skills @Eilon! 😄

**Comment by mmanela** [comment]
@Eilon I think you are looking at this all wrong. I think of it more as referring to the incredible wingspan of the timeless Mr. T.

![image](https://cloud.githubusercontent.com/assets/304410/20722958/73b0558e-b61d-11e6-874a-807115a3852a.png)

**Comment by Eilon** [comment]
@mmanela I can't believe I didn't think of that!

**Comment by terrajobst** [comment]
Mr T is already taken by MRT, the managed runtime :-)

**Comment by filipw** [comment]
Coincidentally `Spant` in German is a ship frame, which, after all, "gives the ship its shape and strength". 

Very fitting (pun intended).

**Comment by richardszalay** [comment]
Related: can we use `OhTheHugeSpanOfTException` when the requested size is too large?

**Comment by omariom** [comment]
![image](https://cloud.githubusercontent.com/assets/1781701/20729695/32c31382-b694-11e6-94a9-ed859ffad79d.png)

**Comment by CaptainEasyCode** [comment]
Spant<T>s - the superior pants.

**Comment by KrzysztofCwalina** [comment]
Now that Span<T> moved to the corefx repo, we lost the ability to have logos for individual types, and so we cannot have a logo for Span<T>. 

Corefx does support one giant logo for all types. In this logo, some special types got 24 pixels (logolets) to use for their individual logo needs. You might want to talk to all the .net stakeholders, convince them that Span<T> deserves a logolet, get the pixels allocated, and then come up with a way to represent a manatee in 24 pixels. 

It's just better like this. The logo architecture was painstakingly designed over the last few years, and we concluded many logos are bad. Yes, the architectural we settled on results in blank reserved spaces in the giant logo, but we have plans to throw NotSupportedException when this part of the logo is decoded using our image manipulation libraries. Also, it might be hard to convince the stakeholders that your type deserves the logolet, so your type might not end up in the logo for many years, or ever.
